src/psyclone/domain/lfric/lfric_loop_bounds.py

```diff
@@ -36,25 +36,17 @@
 # Modified J. Henrichs, Bureau of Meteorology
 # Modified A. B. G. Chalk and N. Nobre, STFC Daresbury Lab
 
-''' This module implements the PSyclone Dynamo 0.3 API by 1)
-    specialising the required base classes in parser.py (KernelType) and
-    adding a new class (DynFuncDescriptor03) to capture function descriptor
-    metadata and 2) specialising the required base classes in psyGen.py
-    (PSy, Invokes, Invoke, InvokeSchedule, Loop, Kern, Inf, Arguments and
-    Argument). '''
+''' This module Handles all variables required for specifying loop limits within a
```

Comment:
Please reword to say it provides the `LFRicLoopBounds` class that .... for the LFRic PSy layer.